server/database/migrations/20190529204216-create-author.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable */
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Authors', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      fullname: {
+        type: Sequelize.STRING,
+        allowNull: false
+      }
```

Comment:
Your Authors migration file should contain the 'createdAt' and 'upatedAt' fields that Sequelize provides by default. Don't remove them.

Reply:
> Your Authors migration file should contain the 'createdAt' and 'upatedAt' fields that Sequelize provides by default. Don't remove them.

I will add them